src/components/Picklist/helpers/isOptionVisible.js

```diff
@@ -0,0 +1,8 @@
+export default function isOptionVisible(el, container) {
+    const { top: elTop, bottom: elBottom } = el.getBoundingClientRect();
+    const { top: containerTop, bottom: containerBottom } = container.getBoundingClientRect();
+    return (
+        Math.floor(elTop) >= Math.floor(containerTop) &&
+        Math.ceil(elBottom) <= Math.ceil(containerBottom)
```

Comment:
Why use Math.floor and Math.ceil?

Reply:
because getboundingclientrect returns floats and sometimes an small fraction casues failure